airflow/configuration.py

```diff
@@ -153,6 +153,7 @@ def __init__(self, default_config=None, *args, **kwargs):
             self.airflow_defaults.read_string(default_config)
 
         self.is_validated = False
+        self.optionxform = str
```

Comment:
Instead of doing this inside `init` could we do `optionxform = str` at the class level? And a short comment explaining what this does please.

Reply:
ConfigParser makes every config parameter lower case by default. Here I am overriding the default behaviour of the `optionxform` function which is `optionstr.lower()`. Line 875 here - https://github.com/python/cpython/blob/master/Lib/configparser.py. So I think it's not possible to do it at class level.

Reply:
I will change it to be at the class level.